utils/meta_parameters.rb

```diff
@@ -53,7 +53,7 @@ def check_for_null(expr, incl = true)
     end
   end
 
-  def sanitize_expression(expr, checks = check_for_null(expr, false), default = 0)
```

Comment:
what? You can pass as arguments to a function used a default parameter another parameter.  mind blow

Reply:
I know right!!!